bedrock/3_EntityWorkflow/workflows.py

```diff
@@ -0,0 +1,174 @@
+import asyncio
+from datetime import timedelta
+from temporalio import workflow
+from collections import deque
+from typing import List, Tuple, Deque, Optional
+from dataclasses import dataclass
+
+with workflow.unsafe.imports_passed_through():
+    from activities import prompt_bedrock
+
+
+@dataclass
+class BedrockParams:
+    conversation_summary: Optional[str] = None
+    prompt_queue: Optional[Deque[str]] = None
+
+
+@workflow.defn
+class EntityBedrockWorkflow:
+    def __init__(self) -> None:
+        # List to store prompt history
+        self.conversation_history: List[Tuple[str, str]] = (
+            []
+        )
+        self.prompt_queue: Deque[str] = deque()
+        self.conversation_summary: Optional[str] = None
+        self.continue_as_new_per_turns: int = 6
+        self.chat_ended: bool = False
+
+    @workflow.run
+    async def run(
+        self,
+        params: BedrockParams = None,
```

Comment:
This will not pass type checking. You can't assign `None` to a non-optional parameter.